src/js/translator-mode.js

```diff
@@ -108,7 +107,7 @@ export function macroSystem(initialMacros) {
     for (const { 0: text, index } of split) {
       const replacement = getMacro(text.substring(1));
       replaced.push(input.substring(lastIdx, index));
-      replaced.push(replacement || text);
```

Comment:
Is eslint sure about this one? Is it correct to not push `text` if `replacement` is an empty string?